bin/setup.js

```diff
@@ -0,0 +1,32 @@
+/* eslint no-console: 0 */
+var fs = require('fs')
+var resolve = require('path').resolve
+var join = require('path').join
+var cp = require('child_process')
+var os = require('os')
+
+var root = resolve(__dirname, '..')
+var docs = resolve(__dirname, '..', 'docs')
+var isWindows = os.platform().startsWith('win')
+var npmCmd = isWindows ? 'npm.cmd' : 'npm'
+var rmrf = isWindows ? 'if exist "node_modules" (rd /s /q node_modules)' : 'rm -rf node_modules'
+
+cp.execSync('cls', { env: process.env, stdio: 'inherit' })
```

Comment:
This is not a GNU Tool... We use `clear` but I do not see a need for clearing the console.

Reply:
Probably don't, this is more of a personal habit. Will take it out